### PR TITLE
Swapna_Assignment_19_Done_60+10+5+5+10

### PR DESCRIPTION
**Pull request by SwapnaWJun20**
image
image
image
image
image

### CODE REVIEW

File: src/swapnaWJun20/PrimeOperations.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                  a) Given number is prime or not. 
+                  b) Print total count of prime numbers between 2->100.
+                  c) Find sum of all prime numbers between 2->100
+                  d) Find avg of prime numbers between 2->100*/
+
+package swapnaWJun20;
+
+public class PrimeOperations {
+
+	void totalCount(int a, int b) {
```
Comment (KrishnaKTechnocredits) [inline src/swapnaWJun20/PrimeOperations.java]: Method name is not expected like this... findPrimeNumberInGivenRange(first,last)

File: src/swapnaWJun20/PrimeOperations.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		int sum = 0;
+		int count = 0;
+		for (int start = a; start < b; start++) {
+			boolean flag = false;
+			for (int index = 2; index <= Math.sqrt(start); index++) {
+				if (start % index == 0) {
```
Comment (KrishnaKTechnocredits) [inline src/swapnaWJun20/PrimeOperations.java]: Why are you writing whole logic here, call isPrime() method.

**Review by KrishnaKTechnocredits — CHANGES_REQUESTED** [review]
.

File: src/swapnaWJun20/PrimeOperations.java (hunk 1 above)
Comment (SwapnaWJun20) [reply src/swapnaWJun20/PrimeOperations.java]: @KrishnaKTechnocredits Changes done as per the comments.

File: src/swapnaWJun20/PrimeOperations.java (hunk 2 above)
Comment (SwapnaWJun20) [reply src/swapnaWJun20/PrimeOperations.java]: @KrishnaKTechnocredits  Changes done as per the comments.

**Pull request merged**